libraries/Ethernet/src/utility/ethernetif.c

```diff
@@ -128,9 +128,11 @@ void HAL_ETH_MspInit(ETH_HandleTypeDef *heth)
     }
   }
 
+#ifdef ETH_INPUT_USE_IT

```

Comment:
why is ETH_INPUT_USE_IT introduced here ? Is it working both with Interrupts and without interrupts?